### PR TITLE
ENH- making the scanning status indicator into a button.

### PR DESCRIPTION
**Pull request by rksh**
from user requests

### CODE REVIEW

**Comment by bobzilladev** [comment]
only thought is hitting that by accident would be sad for people hours later, might want a confirmation dialog?

**Comment by rksh** [comment]
interesting point - trade-off between "quick button" and "oops" seems complicated. could warn the first time. or make it preference-able...

**Comment by bobzilladev** [comment]
maybe quick press has a confirm dialog, which also explains that a long-press does it without a dialog?

**Comment by rksh** [comment]
the votes are in, and it's a 3-way tie! :(
https://wigle.net/phpbb/viewtopic.php?f=1&t=2484
waiting for more input.

**Comment by bobzilladev** [comment]
I like the "dialog with checkbox to not alert again" pattern you suggested. Seems to make everyone happy.

**Comment by rksh** [comment]
implemented as discussed

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ListFragment.java [hunk 1]
```diff
@@ -150,6 +152,10 @@
     public static final String ANONYMOUS = "anonymous";
     public static final String WIFI_LOCK_NAME = "wigleWifiLock";
 
+    public static final String QUCIK_SCAN_UNSET = "UNSET";
+    public static final String QUCIK_SCAN_DO_NOTHING = "DO_NOTHING";
+    public static final String QUCIK_SCAN_PAUSE = "PAUSE";
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ListFragment.java]: is QUCIK a reference i don't get?

**Comment by rksh** [comment]
closing because rebase crap is in the PR (and shouldn't be)

**Pull request closed without merging**